### ISSUE TITLE
not able to find GetxService when app goes to background....

### ISSUE THREAD

**Issue by rmControls** [jonataslaw/getx#1837]
getting error message `GetxService ` not found when app goes to background .... you get following error message when app only in background...
`
"MessageService" not found. You need to call "Get.put(MessageService())" or "Get.lazyPut(()=>MessageService())"`

 it works perfectly when app in Foreground.

Please check...

**Comment by saravananmnm** [comment]
Yes. Same issue here.

**Comment by rmControls** [comment]
@jonataslaw can you please help us with this issue...??

**Comment by saravananmnm** [comment]
@jonataslaw @jasonlaw @lsm Any workarounds for this.?

**Comment by MrCsabaToth** [comment]
So I was running into a service disappearing in my app which was not disappearing before. The Smart Management decided to ditch that service. I have a feeling that maybe it has to do something with the fact that I started to obey Android Studio linting suggestions and applied a lot of `const` keywords and alike and maybe that showed some memory objects more discardable than before, but that is just really a guess.

The bottom line is - try to tell `Get` to keep your service in the memory by utilizing the `permanent` named parameter of `put`, like so: `Get.put(MessageService(), permanent: true)`. Of course this has some consequences such as you need to take control when you want to discard that service manually, so you have to command the lifecycle of the service on your own.

**Comment by disonwei** [comment]
https://github.com/jonataslaw/getx/issues/3029     The same issue

**Comment by deverlex** [comment]
Same issue, haizzz
